src/dat/controllers/Controller.js

```diff
@@ -54,6 +54,22 @@ class Controller {
     this.__onFinishChange = undefined;
   }
 
+  /**
+  * Hides the controller on it's parent GUI.
+  */
+  hide() {
+    this.domElement.parentNode.parentNode.style.display = 'none';
+    return this;
+  }
+
+  /**
+  * Shows the controller on it's parent GUI.
+  */
+  show() {
+    this.domElement.parentNode.parentNode.style.display = '';
+    return this;
+  }
```

Comment:
I'm not sure about having a per-controller show/hide feature... Do you mind putting this part in a separate PR?